_posts/2017-04-22-Annoying-Mastodon-Users-with-a-Bot.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Annoying Mastodon Users with a Bot
+layout: post
+permalink:
+author: bobby_brb3 
+date: 2017-04-22
+---
+
+Because of the recent influx of Japanese users on Mastodon, I wrote a bot to
+translate their toots to English. Unfortunately, this annoyed a large portion of
+the Mastodon user base. If you're interested in learning a bit about writing a
```

Comment:
Ah yea, I had to trim this down to fit into the 50 word summary thing. I should have just wrote a different summary, I think. Ok, I'll work on that.